packages/e2e-tests/helpers/testsuites.ts

```diff
@@ -1,25 +1,166 @@
+  const pane = getTestSuitePanel(page);
+
+  let isVisible = await pane.isVisible();
+
+  if (!isVisible) {
+    await getByTestName(page, "ToolbarButton-CypressPanel").click();
+  }
+}
+
+export function getTestRows(page: Page) {
+  return getByTestName(page, "TestRecordingTreeRow");
 }
 
-export async function getCypressLogo(page: Page) {
-  return page.locator('[data-test-name="ToolbarButton-CypressPanel"]');
+export function getCypressLogo(page: Page) {
+  return getByTestName(page, "ToolbarButton-CypressPanel");
 }
 
-export async function getTestRowChevron(row: Locator) {
+export function getTestRowChevron(row: Locator) {
   return row.locator(":scope", { hasText: "chevron_right" });
 }
 
-export async function getTestSections(row: Locator) {
-  return row.locator('[data-test-name="TestSection"]');
+export function getTestRecordingTrees(page: Page) {
+  return getByTestName(page, "TestRecordingTree");
+}
+
+export function getTestSections(row: Locator) {
+  return getByTestName(row, "TestSection");
+}
+
+export function getTestCaseSteps(row: Page | Locator) {
+  return getByTestName(row, "TestSectionRow");
+}
+
+export function getSelectedTestCase(row: Page | Locator) {
+  return row.locator('[data-test-id="TestRecordingPanelBody"]');
+}
+
+export function getTestSuiteResult(page: Page) {
+  return getByTestName(page, "TestSuiteResult");
+}
+
+export function getTestSuiteResultsPassedCount(page: Page) {
+  return getByTestName(page, "TestSuiteResultsPassedCount");
+}
+
+export function getTestSuiteResultsFailedCount(page: Page) {
+  return getByTestName(page, "TestSuiteResultsFailedCount");
+}
+
+export function getTestSuiteResultsSkippedCount(page: Page) {
+  return getByTestName(page, "TestSuiteResultsSkippedCount");
+}
+
+export function getTestRecordingBackButton(page: Page) {
+  return getByTestName(page, "TestRecordingBackButton");
 }
 
-export async function getTestCaseSteps(row: Locator) {
-  return row.locator('[data-test-name="TestSectionRow"]');
+export function getTestSuiteDate(page: Page) {
+  return getByTestName(page, "TestSuiteDate");
 }
 
-export async function getSelectedTestCase(row: Pick<Locator, "locator">) {
-  return row.locator('[data-test-id="TestItemPanelBody"]');
+export function getTestSuiteUser(page: Page) {
+  return getByTestName(page, "TestSuiteUser");
+}
+
+export function getTestSuiteDuration(page: Page) {
+  return getByTestName(page, "TestSuiteDuration");
+}
+
+export function getTestSuiteBranch(page: Page) {
+  return getByTestName(page, "TestSuiteBranch");
+}
+
+export function getErrorRows(page: Page) {
+  const steps = getTestCaseSteps(page);
+  return steps.filter({
+    has: page.locator(`[data-status="error"]`),
+  });
+}
+
+export function getTestStepBeforeAfterButtons(page: Page) {
+  return getByTestName(page, "TestBeforeAfter");
+}
+
+export function getUserActionEventDetails(page: Page) {
+  return getByTestName(page, "UserActionEventDetails");
+}
+
+export async function getKeyValueEntry(
```

Comment:
nit (maybe for follow up, not blocking: Some of these seem like they may be more appropriately put in the object-inspector test helper